src/Symfony/Component/Security/Http/HttpUtils.php

```diff
@@ -175,4 +175,21 @@ public function generateUri($request, $path)
 
         return $url;
     }
+
+    public static function getForRuntime(UrlGeneratorInterface $urlGenerator = null, $urlMatcher = null, string $cookieSecure = null, string $domainRegexp = null)
```

Comment:
`create()`?

Reply:
> Instead, I'd suggest adding a static factory method to handle the runtime creation of instances

no  name suggestion :P i suggest at least create over get, as multiple calls dont "get" the same instance, but creates 2 new ones. "ForRuntime" is obvious to me, hence `static create()`

Reply:
or `constructForRuntime`?